src/ui/geonu.tsx

```diff
@@ -29,8 +29,20 @@ import {
   LUNAR_MANTLE_MASS,
 } from "../mantle/geophysics";
 
-const {K40, Th232, U235, U238} = ElementsUI
+import {
+  layerMasses,
+  layerGeoResponse,
+} from "../mantle/PREM";
 
+const massFunc = (radBot, radTop) => {
+  return layerMasses.slice( radBot * 10, radTop * 10 ).reduce((massSum, currentMass)=>massSum + currentMass)
```

Comment:
The crash is being caused here when the radBot and radTop have the same value, this causes the slice to be empty so reduce throws. This can be solved by setting an initial value (0?) to the reduce function in both here (massFunc) and geoResponseFunc.

```javascript
return layerMasses.slice( radBot * 10, radTop * 10 ).reduce((massSum, currentMass)=>massSum + currentMass, 0) // see the added 0 argument to reduce 
```